spec/ssl/gen_certs.sh

```diff
@@ -35,7 +39,7 @@ commonName_default             = mysql2gem.example.com
 
 # Generate a set of certificates
 openssl genrsa -out ca-key.pem 2048
-openssl req -new -x509 -nodes -days 3600 -key ca-key.pem -out ca-cert.pem -batch -config ca.cnf
+openssl req -new -x509 -nodes -extensions v3_ca -days 3600 -key ca-key.pem -out ca-cert.pem -batch -config ca.cnf
```

Comment:
In downstream we have simply appended the following: `-addext basicConstraints=critical,CA:TRUE,pathlen:1`

Not sure which is the best/preferred way.